types/reactstrap/lib/Input.d.ts

```diff
@@ -41,5 +41,5 @@ export interface InputProps extends React.InputHTMLAttributes<HTMLInputElement>
   cssModule?: CSSModule;
 }
 
-declare const Input: React.StatelessComponent<InputProps>;
+declare const Input: React.ComponentClass<InputProps>;
```

Comment:
Wouldn't extending `React.Component` be more accurate instead of `React.ComponentClass`?

Reply:
Yes, I suppose so. Changed.